docs/source/user_manual/chaco_tutorial.rst

```diff
@@ -1290,7 +1290,8 @@ Final words
 This concludes this tutorial. You can download :download:`a PDF version of the slides <../resources/scipy08_tutorial.pdf>`
 For further information, visit the :ref:`user_guide`. You can find the examples
 for this tutorial in the :file:`examples/tutorials/scipy2008/` directory of the
-Chaco source code. You can browse it online in the `GitHub repository <https://github.com/enthought/chaco/tree/master/examples/tutorials>`_
+Chaco source code. You can browse it online in the
```

Comment:
same comment as in traitsui/pyface regd not linking to code in the main/master branches - and instead linking to specific versions